Library/Homebrew/test/cli/parser_spec.rb

```diff
@@ -210,4 +210,25 @@
       expect { parser.parse(["--switch-b"]) }.to raise_error(RuntimeError, /Arguments were already parsed!/)
     end
   end
+
+  describe "test argv extensions" do
+    subject(:parser) {
+      described_class.new do
+        switch "--foo"
+        flag   "--bar"
+        switch "-s"
+        switch :verbose
+      end
+    }
+
+    it "#options_only" do
+      parser.parse(["--foo", "--bar=value", "-v", "-s", "a", "b", "cdefg"])
```

Comment:
`long` is given a higher preference compared to `short` and as a side effect when `-v` is passed on CLI, `args.options_only` return `--verbose`. I hope this doesn't bite us going forward ?

Reply:
That seems fine 👍